custom/icds_reports/models/views.py

```diff
@@ -1392,3 +1392,43 @@ class Meta(object):
         app_label = 'icds_reports'
         managed = False
         db_table = 'bihar_demographics_view'
+
+
+class KPIAPIView(models.Model):
+    """
+    Contains rows for KPI API.
+    """
+    district_id = models.TextField(blank=True, null=True)
+    district_name = models.TextField(blank=True, null=True)
+    district_site_code = models.TextField(blank=True, null=True)
+    state_id = models.TextField(blank=True, null=True)
+    state_name = models.TextField(blank=True, null=True)
+    state_site_code = models.TextField(blank=True, null=True)
+    aggregation_level = models.IntegerField(blank=True, null=True)
+    district_map_location_name = models.TextField(blank=True, null=True)
```

Comment:
i think you can remove these map_location_names

Reply:
removed